values.yaml

```diff
@@ -543,11 +543,13 @@ postgresql:
 
 # externalRedis -- External Redis configuration. To use an external Redis instead of the automatically deployed redis chart: set redis.enabled to false then uncomment and configure the externalRedis connection URL (e.g. redis://user:pass@host:6379/database).
 externalRedis:
-# externalRedisSecret -- Read external Redis configuration from a secret. This should point at a secret file with a single key which specifyies the connection string.
-externalRedisSecret:
-  {}
-  # name: redash-redis
-  # key: connectionString
+  host: ""      # The name of the Kube service you created elsewhere, pointing to your Redis master.
+  database: ""  # Stays empty in the case of MemoryStore
+  port: 6379
+  auth:
+    enabled: false  # Whether or not to use authentication
+    secretName: ""  # Name of the secret where the external Redis's auth string is to be found
+    secretKey: "redisAuthString"
```

Comment:
Where is it documented or checked that the key name is the one hardcoded here when creating the secret? I'm just making sure we are not missing some parts where this could be missed.